lib/crowdin/adapter.rb

```diff
@@ -188,7 +190,7 @@ def files_by_model_and_id(model_name, id)
       file_name = file_name(model_name, id)
       file_base_name = File.basename(file_name, ".*")
 
-      @client.find_file_by_name(file_base_name)
+      [@client.find_file_by_name(file_base_name)]
```

Comment:
could we change the name of this function to `file_by_model_and_id`, and keep the return type as just a single file? There's no possibility as it's currently written to return multiple. Then in `cleanup_translation` you don't need the `flatten`, and i think we should do the array construction in `delete_source_files_for_model`, rather than at this level. Thoughts?

Reply:
makes sense, I didn't look fully into the `client.find_file_by_name` at the time to see that it only returns the first match. I think theres an implicit assumption here that there will only be one source file for each model, this should always be the case right? at least for the foreseeable future